### ISSUE TITLE
Reviews does not appear to find .env file in current directory?

### ISSUE THREAD

**Issue by larsks** [apoclyps/reviews#580]
**Describe the bug**

I have added a `.env` file to the top level of a local repository. If I run `reviews` from within that directory, it does not appear to read the `.env` file.

**To Reproduce**

```
$ cd ~/projects/kola
$ git rev-parse --show-toplevel
/home/lars/projects/kola
$ cat > .env <<EOF
REVIEWS_GITHUB_REPOSITORY_CONFIGURATION=larsks/kola
GITHUB_TOKEN=gho_...
GITHUB_USER=larsks
EOF
$ reviews config --show
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Name                                    ┃ Value                                                   ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ GITHUB_TOKEN                            │                                                         │
│ GITHUB_USER                             │                                                         │
│ GITHUB_URL                              │ https://api.github.com                                  │
│ GITLAB_TOKEN                            │                                                         │
│ GITLAB_USER                             │                                                         │
│ GITLAB_URL                              │ https://gitlab.com                                      │
│ REVIEWS_PATH_TO_CONFIG                  │ None                                                    │
│ GITHUB_DEFAULT_PAGE_SIZE                │ 100                                                     │
│ REVIEWS_DELAY_REFRESH                   │ 60                                                      │
│ REVIEWS_GITHUB_REPOSITORY_CONFIGURATION │ apoclyps/reviews                                        │
│ REVIEWS_GITLAB_REPOSITORY_CONFIGURATION │ 27629846:apoclyps/reviews                               │
│ REVIEWS_LABEL_CONFIGURATION             │ blocked/orange, docker/blue, security/red, python/green │
└─────────────────────────────────────────┴─────────────────────────────────────────────────────────┘
```

However, if I explicitly load the `.env` file (using e.g. `dotenv`) it works as expected:

```
$ dotenv run reviews config --show
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Name                                    ┃ Value                                                   ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ GITHUB_TOKEN                            │ gho_...                                                 │
│ GITHUB_USER                             │ larsks                                                  │
│ GITHUB_URL                              │ https://api.github.com                                  │
│ GITLAB_TOKEN                            │                                                         │
│ GITLAB_USER                             │                                                         │
│ GITLAB_URL                              │ https://gitlab.com                                      │
│ REVIEWS_PATH_TO_CONFIG                  │ None                                                    │
│ GITHUB_DEFAULT_PAGE_SIZE                │ 100                                                     │
│ REVIEWS_DELAY_REFRESH                   │ 60                                                      │
│ REVIEWS_GITHUB_REPOSITORY_CONFIGURATION │ larsks/kola                                             │
│ REVIEWS_GITLAB_REPOSITORY_CONFIGURATION │ 27629846:apoclyps/reviews                               │
│ REVIEWS_LABEL_CONFIGURATION             │ blocked/orange, docker/blue, security/red, python/green │
└─────────────────────────────────────────┴─────────────────────────────────────────────────────────┘

```

**Expected behaviour**

I expected `reviews` to read the `.env` file.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Fedora (Linux)
 - Version [e.g. 22] 37

**Comment by apoclyps** [comment]
Hi @larsks 👋 

Thanks for raising this issue. I will spend some time on it over the next few days however if you want to attempt to solve it and submit a pull request, I would be more than happy to review it.

From the issue you shared above, we should be defaulting to AutoConfig() on L9 within `reviews/config/settings.py` in 
https://github.com/apoclyps/reviews/blob/a9f46850539694dcd8f069a35e297ce497450aee/reviews/config/settings.py#L5-L10

This makes use of [python-decouple](https://pypi.org/project/python-decouple/)  which uses `AutoConfig()` to do the following:

> This is a lazy Config factory that detects which configuration repository you’re using.
>
> It recursively searches up your configuration module path looking for a settings.ini or a .env file.
>
> Optionally, it accepts search_path argument to explicitly define where the search starts.

Depending on where AutoConfig runs from, it may be possible it's running from where `reviews` is installed and not the current working directory. To support the current working directly this may need to be updated to specify the `search_path=os.getcwd()`

**Comment by larsks** [comment]
With `reviews` installed in `/home/lars/.local/pipx/venvs/reviews/lib/python3.11/site-packages/reviews`, and run from `/home/lars/projects/kola`, the search path for `.env` is:

```
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/lib64/python3.11/site-packages/reviews/config/.env", 0x7ffe0d108070, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/lib64/python3.11/site-packages/reviews/.env", 0x7ffe0d108070, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/lib64/python3.11/site-packages/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/lib64/python3.11/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/lib64/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/reviews/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/venvs/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/pipx/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.local/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/lars/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
71595 newfstatat(AT_FDCWD, "/home/.env", 0x7ffe0d1080d0, 0) = -1 ENOENT (No such file or directory)
```

...which is of course silly. I think just passing `.` to `AutoConfig()` will probably do the right thing.

**Comment by apoclyps** [comment]
Thank you for your pull request @larsks - this will be made available in the 0.5.1 release of `reviews`

relates to:
- https://github.com/apoclyps/reviews/pull/585
